### PR TITLE
Implement Add Grade Command

### LINKED ISSUE #278: Implement Add Grade Command
so that grades can be input to the Grade Tracker

### PR DESCRIPTION
**Pull request by murtubak**
fixes #278

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/CliSyntax.java [hunk 1]
```diff
@@ -15,4 +15,5 @@
     public static final Prefix PREFIX_ADD_NEW_ASSIGNMENT = new Prefix("a/");
     public static final Prefix PREFIX_PERCENTAGE_ASSIGNMENT = new Prefix("%/");
     public static final Prefix PREFIX_RESULT_ASSIGNMENT = new Prefix("r/");
+    public static final Prefix PREFIX_GRADE = new Prefix("ag/");
```
Comment (MatthiasLHK) [inline src/main/java/seedu/address/logic/parser/CliSyntax.java:18]: I think the prefix should just be called g/, as Im assuming that we add the "addgrade" command in front, so no need to specify the "a" in the prefix.

File: src/main/java/seedu/address/model/module/Module.java [hunk 2]
```diff
@@ -114,6 +115,21 @@ public Module addAssignment(Assignment assignment) {
...
+     * @return Module a new module with the grade added.
+     */
+    public Module addGrade(Grade grade) {
+        if (Grade.isValidGrade(grade.gradeResult)) {
+            gradeTracker.setGrade(grade);
+            return new Module(name, zoomLink, gradeTracker);
+        } else {
+            return this;
+        }
+    }
```
Comment (MatthiasLHK) [inline src/main/java/seedu/address/model/module/Module.java:131]: I thought we decided on using an association class instead? Im not too sure also. KIV

File: src/main/java/seedu/address/logic/parser/CliSyntax.java (hunk 1 above)
Comment (murtubak) [reply src/main/java/seedu/address/logic/parser/CliSyntax.java]: Ok, sounds good.

**Review by MatthiasLHK — APPROVED** [review]
LGTM

**Pull request merged**